genienlp/arguments.py

```diff
@@ -521,7 +521,7 @@ def parse_argv(parser):
 
     # token classification task args
     parser.add_argument('--num_labels', type=int, help='num_labels for classification tasks')
-    parser.add_argument('--ner_domains', nargs='+', type=str, help='domains to use for CrossNER task')
+    parser.add_argument('--crossner_domains', nargs='+', type=str, help='domains to use for CrossNER task')
```

Comment:
I don't understand, we're going in circles. Why is this a separate argument?

Reply:
Crossner dataset has multiple subfolders. Each containing train/eval/test splits per domain. This argument indicates which domain we wanna do an experiment on. This argument has nothing to do with NED.

Reply:
I feel like that should be a pipeline problem then, but ok at least it's a scoped argument specific to the task, like the almond_* stuff.